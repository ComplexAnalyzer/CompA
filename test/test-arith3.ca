cx foo(float a)
{ 
  cx c;
  c= (3.2,3.4);
  c[0]=a;
  return c ;
}

int main()
{
  cx res;
  res = foo(3.2);
  print(res);
  return 0;
}