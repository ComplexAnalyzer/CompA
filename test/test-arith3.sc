def foo(a)
{
  return a;
}

def main()
{
  a = 42;
  a = a + 5;
  print(a);
  return 0;
}
