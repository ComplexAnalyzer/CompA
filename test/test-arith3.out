(3.200000,3.400000)
